intro-OpenStack/ch_LaunchInstances.tex

```diff
@@ -0,0 +1,272 @@
+  selected for launching an instance. For example, while creating an
+  image, if you have entered the value in the Minimum RAM (MB) field
+  as 2048, then on selecting the image, the default flavor is
+  \textbf{m1.small}.  If a '!' warning sign is displayed next to a
+  resource for one of the flavors, that means that this flavor would
+  exceed the project's quota for that resource, and therefore is not
+  available.
+
+\item[Networks] tab. Add one or more networks to the instance.
+
+\item[Network Ports] tab. Activate the ports that you want to assign to
+  the instance.
+\item[Security Groups] tab. Activate the security groups that you want
+  to assign to the instance.
+
+  Security groups are a kind of cloud firewall that define which
+  incoming network traffic is forwarded to instances.
+
+  The default security group is assigned to the instance
+  automatically.
+\item[Key Pair] tab. Specify a key pair.
+
+  If the image uses a static root password or a static key set
+  (neither is recommended), you do not need to provide a key pair to
+  launch the instance.
+\item[Configuration] tab. Specify a customization script that runs
+  after your instance launches.
+
+\item[Server Groups] tab.
+
+\item[Scheduler Hints] tab.
+
+\item[Metadata] tab. Add Metadata items to your instance.
+\end{description}
+
+\item Click Launch Instance.
+\end{enumerate}
+
+The instance starts on a compute node in the cloud.
+
+\strong{Note:} If you did not provide a key pair, security groups, or
+rules, users can access the instance only from inside the cloud
+through VNC. Even pinging the instance is not possible without an ICMP
+rule configured.
+
+You can also launch an instance from the Images or Volumes category when
+you launch an instance from an image or a volume respectively.
+
+When you launch an instance from an image, OpenStack creates a local
+copy of the image on the compute node where the instance starts.
+
+For details on creating images, see
+\href{https://docs.openstack.org/image-guide/create-images-manually.html}{\emph{Creating
+images manually}} in the \emph{OpenStack Virtual Machine Image Guide}.
+
+\section{Connect to your instance using SSH}\label{connect-to-your-instance-using-ssh}
+
+\subsection*{Obtain a public ip}
+Only a project's floating ip's can be reached from outside of the
+OpenStack network.  One way to obtain a public ip for an instance, is
+to associate a floating ip to it, as detailed in section
+\ref{allocate-a-floating-ip-address-to-an-instance} on page
+\pageref{allocate-a-floating-ip-address-to-an-instance}.  However,
+only a limited number of floating ip's are available to each cloud
+project, and this approach uses one ip for every instance you want to
+connect to.  A more economical approach, discussed in the next
+section, is to use different ports of a single floating ip to reach
+different instances.
+
+\subsubsection{Floating ip port forwarding}
```

Comment:
This should be the recommended option in our case.
Maybe we should move the `{allocate-a-floating-ip-address-to-an-instance}` section also here  as a new subsection (advanced option)